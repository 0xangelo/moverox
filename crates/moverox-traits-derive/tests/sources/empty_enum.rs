use moverox_traits::MoveDatatype;

#[derive(MoveDatatype)]
pub enum Empty {}

fn main() {}
