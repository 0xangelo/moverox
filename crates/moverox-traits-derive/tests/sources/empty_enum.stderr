error: A Move 'enum' must define at least one variant
 --> tests/sources/empty_enum.rs:3:10
  |
3 | #[derive(MoveDatatype)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `MoveDatatype` (in Nightly builds, run with -Z macro-backtrace for more info)
